dev/format_code.sh

```diff
@@ -1,14 +1,35 @@
 #!/usr/bin/env bash
+set -eo pipefail
 
 # Formats all files which differ from their state on the master branch with
 # clang-format.
 #
 # Usage:
 #   - Make sure the executables "clang-format" and "git" are available in PATH.
 #   - Run the command "./dev/format_code.sh" in the root of the repository.
+#   - To run clang-format in a docker-container, use the "--docker" parameter.
 
 echo "Formatting modified files with clang-format..."
 
+DOCKER_IMAGE=librepcb/clang-format:6
+if [ "$1" == "--docker" ]; then
+  if [ "$(docker images -q $DOCKER_IMAGE | wc -l)" == "0" ]; then
+    echo "Building clang-format container..."
+    cd clang-format
+    docker build . -t librepcb/clang-format:6
+    cd ..
+  fi
+
+  echo "[Re-running format_code.sh inside Docker container]"
+  docker run --rm -t -i \
```

Comment:
Maybe better add `--user $(id -u):$(id -g)` to improve security and avoid possible permission issues with the mounted volume?

Reply:
Yeah, good point.